### ISSUE TITLE
parseSize issue

### ISSUE THREAD

**Issue by tomalex0** [hacksparrow/node-easyimage#68]
I'm getting `sizeArray` as null when its called like `parseSize("34.6KB");`

Are you facing this issue?

https://github.com/hacksparrow/node-easyimage/blob/master/easyimage.js#L32

```
var parseSize = function(sizeString) {
        console.log(sizeString,'sizeString')

		var unit = {
			B: 1,
			KB: 1000,
			MB: 1000000,            // =1000^2
			GB: 1000000000,         // =1000^3
			TB: 1000000000000       // =1000^4
		};

		var rx = /^(\d\.?\d*)([KMGT]?B)$/;  // regex for extract the float value and its unit
               var sizeArray = rx.exec(sizeString);

        

		return parseFloat(sizeArray[1]) * unit[sizeArray[2]];
	};
```

**Comment by cruzlutor** [comment]
Me too, I'm in windows, i had to change 

```
info.size    = parseSize(temp[4]); 
```
to
```
info.size    = temp[4];
```

But is not a good solution

**Comment by tomalex0** [comment]
How about changing regex to
`var rx = /^(\d*.?\d*)([KMGT]?B)$/;` instead of
`var rx = /^(\d\.?\d*)([KMGT]?B)$/;`

**Comment by hacksparrow** [comment]
Please run some tests and send a PR.

**Comment by tomalex0** [comment]
Updated test cases here
http://runnable.com/VVWul8dfxdANgDif/file-size-regex-split-for-node-js

It breaks for  values like 1KB, 12KB, 100KB ....

**Comment by pronebird** [comment]
Same issue.

**Comment by hacksparrow** [comment]
Fixed by PR #69
